### DISCUSSION TITLE
How to make a bot float with the flow of water?

### DISCUSSION THREAD

**Discussion by Fridrih909** [ORelio/Minecraft-Console-Client#2253] [Q&A]
A quick problem. How to make a bot float with the flow of water?

**Comment by BruceChenQAQ** [comment]
MCC does not currently support water-related processing.
See https://github.com/MCCTeam/Minecraft-Console-Client/issues/206

**Reply by Fridrih909** [reply]
but tell me. Is it possible to make the bot run around certain terrain and only mine certain blocks?

**Reply by BruceChenQAQ** [reply]
The latest pre-release version has an AutoDig ChatBot, but it doesn't make the player move.
You can also use ScriptScheduler to make the bot execute the move command at regular intervals to move to a certain location.
Then again, you can have bots sitting on mine carts or boats, using in-game mechanics to get players moving.
